pr_review_list.rb

```diff
@@ -2,6 +2,7 @@
 # frozen_string_literal: true
 
 require 'erb'
+# require 'pry'
```

Comment:
Please remove `pry` rather than just hashing it out - it shouldn't be included in production code